Gems/EMotionFX/Code/Source/Editor/SkeletonModelJointWidget.cpp

```diff
@@ -96,7 +96,10 @@ namespace EMotionFX
 
         if (GetActor())
         {
-            if (!selectedModelIndices.isEmpty())
+            bool isCharacterRootNodeSelected = AZStd::any_of(selectedModelIndices.cbegin(), selectedModelIndices.cend(), [](const auto& index) {
+                return index.data(SkeletonModel::ROLE_NODE_INDEX).value<quint64>() == qulonglong(-1);
+            });
```

Comment:
FYI you can use [AZStd::ranges::any_of](https://github.com/o3de/o3de/blob/development/Code/Framework/AzCore/Tests/AZStd/RangesAlgorithmTests.cpp#L258-L263) to save on a bit of typing